R/plotting.R

```diff
@@ -665,30 +706,6 @@ plotLineageDomainRepeats <- function(query_data, colname) {
 #' }
 #'
 plotLineageHeatmap <- function(prot, domains_of_interest, level = 3, label.size = 8) {
```

Comment:
A fallback is added for color assignment in plotLineageHeatmap, defaulting to black if there are insufficient colors for the unique lineages. This ensures visual consistency without missing colors in the output.